submissions/Jemikalajah-Philip/php/fetch.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require "config.php";
+
+try {
+    session_start();
+    $connection = new PDO ($dsn, $username, $password, $options);
+    $stmt_brands = $connection->query("SELECT * FROM addProducts.brands");
+    $brands = array();
+    $brand_id = array();
+    while ($item = $stmt_brands->fetch()) {
+        array_push($brands, $item['brand_name']);
+        array_push($brand_id, $item['id']);
+    }
+   $mappedBrands = array_combine( $brand_id, $brands);
+    $_SESSION['mappedBrands'] = $mappedBrands;
```

Comment:
If you take heed to the comment on `create.php` you would not need the Session array here or this `fetch.php` file for that matter.